api/src/actions/mapLayersActions.js

```diff
@@ -20,21 +24,25 @@ const getMapLayersWithGeoData = async () => {
   };
 
   const layers = await model.getMapLayers(filter);
-  const layersWithGeoJSONUrl = layers.map((layer) => {
+  const layersWithGeoDataUrl = layers.map((layer) => {
     if (layer.layerType === 'group') {
       const sublayers = layer.layers.map((lr) => ({
         ...lr,
-        geoJSONUrl: geoJSONUrls[lr.geoReferenceId],
+        geoDataUrl: geoDataUrls[lr.geoReferenceId],
+        geoMetadata: geoMetadata[lr.geoReferenceId],
+        format: formats[lr.geoReferenceId],
       }));
       return { ...layer, layers: sublayers };
     }
 
     return {
       ...layer,
-      geoJSONUrl: geoJSONUrls[layer.geoReferenceId],
+      geoDataUrl: geoDataUrls[layer.geoReferenceId],
+      geoMetadata: geoMetadata[layer.geoReferenceId],
```

Comment:
I just return geoMetadata in static layers. Showing them somewhere is not part of this task, correct?

Reply:
looks OK on the frontend, just no info icon is shown because we have no mapLayers metadata there.